cool-core/src/main/java/com/nus/cool/core/cohort/refactor/storage/CohortRet.java

```diff
@@ -0,0 +1,134 @@
+package com.nus.cool.core.cohort.refactor.storage;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map.Entry;
+
+import com.google.common.base.Preconditions;
+import com.nus.cool.core.cohort.refactor.ageSelect.AgeSelection;
+
+/**
+ * Class for Cohort Analysis Result
+ * We consider the Cohort Analysis Result as a x-y Axis
+ * x-Axis is age, y-Axis is value, legend is cohort group
+ * CohortResult class is generated according to the ageselection.
+ */
+public class CohortRet {
+
+    private HashMap<String, Xaxis> cohortToValueList;
+
+    private int min, max, interval;
+
+    private int size;
+
+    public CohortRet(AgeSelection ageSelection) {
```

Comment:
The constructor is only used by `CohortProcessor::init`. Taking the arguments: min, max, and interval explicitly is clearer. (disassociate this class from `AgeSelection` class)

Reply:
done